test/emqx_dashboard_SUITE.erl

```diff
@@ -36,6 +36,8 @@ groups() ->
     ].
 
 init_per_suite(Config) ->
+    ekka_mnesia:start(),
```

Comment:
just `application:load(emqx_storm)`, the ekka would start mnesia when emqx is started